versions.txt

```diff
@@ -1,6 +1,6 @@
 # Format:
 # module:released-version:current-version
 
-google-cloud-compute:0.0.0:0.0.1-SNAPSHOT
-proto-google-cloud-compute-v1:0.0.0:0.0.1-SNAPSHOT
-grpc-google-cloud-compute-v1:0.0.0:0.0.1-SNAPSHOT
\ No newline at end of file
+google-cloud-compute:1.0.0-alpha:1.0.1-alpha-SNAPSHOT
+proto-google-cloud-compute-v1:1.0.0-alpha:1.0.1-alpha-SNAPSHOT
+grpc-google-cloud-compute-v1:1.0.0-alpha:1.0.1-alpha-SNAPSHOT
```

Comment:
For the actual release, you're going to want these to all have `1.0.0-alpha:1.0.0-alpha`. The subsequent PR to bump to the new snapshot will have `1.0.0-alpha:1.0.1-alpha-SNAPSHOT`